public/steelthreads/asciidocs/en/integrating-eventdriven-and-apidriven-applications-introduction.adoc

```diff
@@ -19,13 +15,16 @@ A more structured approach involves:
 
 This walkthrough describes how  to create such an integration, using OpenShift, Red Hat AMQ Online and API connectors in Fuse.
 
-image::st0.png[]
 
-.Prerequisites
+++++
+<img src="/images/st0.png" class="img-responsive" alt="integration">
+++++
+
+//.Prerequisites
 
-* [x] Access to an OpenShift instance with Integreatly components installed
+//* [x] Access to an OpenShift instance with Integreatly components installed
```

Comment:
Is this supposed to be commented out?

Reply:
yeah, that's deliberate, might remove later